README.md

```diff
@@ -33,6 +33,154 @@ The priority items are:
 
 ## Development Setup
 
+To set up the development environment for the Holo Key Manager monorepo, follow these steps:
+
+### Prerequisites
+
+Ensure you have the following installed on your machine:
+
+- Node.js (v16 or higher)
+- pnpm (v7 or higher)
+
+Additionally, request the `.env` file from the codebase maintainer. Once you receive it, ensure that it matches the structure and keys provided in the `.env.example` file.
```

Comment:
I'm not sure we should be documenting a step to request the env file from the maintainer especially as this repo is public and open source. It's probably enough to say that they need an env file like in the example.